charts/k8ssandra/values.yaml

```diff
@@ -81,6 +81,25 @@ cassandra:
    #size:
    #newGenSize:
 
+  # -- Optional cluster-level garbage collection configuration. It can be overridden at
+  # the datacenter level.
+  gc:
+    # -- GC configuration for the CMS collector.
+    cms: {}
```

Comment:
These are good questions. I could have used an empty string instead. I chose `{}` because I have seen that in other charts. To me it indicates that the property has child properties. Will that be clear to the k8ssandra users, particularly those with little or no prior Helm experience? Doubtful. We need to document it.